src/jinja2/environment.py

```diff
@@ -600,6 +600,7 @@ def parse(
         source: str,
         name: t.Optional[str] = None,
         filename: t.Optional[str] = None,
+        preserve_comments: bool = False,
```

Comment:
suggestion: We could make it a keyword-only parameter for easier backward-compatibility maintenance in the future.